Added 2 new coins.

globaltoken opened the pull request:
Hey,

I added 2 new Coins. We have Globaltoken Zhash & Globaltoken Equihash, because we use Multialgorithm.

Greetings,
Pawel

globaltoken committed:
Create glt_zhash.json

globaltoken committed:
Create glt_equihash.json

Gumbees commented:
👍 for the looks. needs tested by one of us. can you add testnet configs so we can throw it up and test? @globaltoken

globaltoken commented:
@nettts The hardfork is not released yet.
I will test it.

globaltoken committed:
Update

- Updated coins.json
- Updated package.json

globaltoken committed:
Added coinbasevaluesupport

Added coinbasevaluesupport

globaltoken committed:
GLT delete hasGetInfo

- Delete hasGetInfo

globaltoken committed:
Merge pull request #1 from s-nomp/master

Merging newest commits

globaltoken committed:
updated GLT configs

globaltoken committed:
added supportsSegwit for GLT

globaltoken committed:
New getaddressinfo cmd

- Changed to getaddressinfo if new coins require it

globaltoken committed:
Validate tAddress via getaddressinfo

globaltoken committed:
zValidate Update

- GLT does not use zaddresses, but lets update the command here to the new format, to make payment processing useable here.

globaltoken committed:
Don't requireShielding for GLT

globaltoken committed:
Don't requireShielding for GLT

sennevb commented:
seems cool, but node-stratum-pool needs to be merged with snomp

egyptianbman commented on package.json:
```diff
@@ -48,7 +48,7 @@
         "redis": "^2.8.0",
         "request": "^2.88.0",
-        "stratum-pool": "git+https://github.com/s-nomp/node-stratum-pool.git"
+        "stratum-pool": "git+https://github.com/globaltoken/snomp-stratum-pool.git"
```
This isn't an option. You'll need to create a PR to https://github.com/s-nomp/node-stratum-pool.git to support this coin and restore the original value.

globaltoken replied:
@egyptianbman I will restore it

globaltoken committed:
Update package.json

globaltoken commented:
Alright, everything is working with this commit. In theory Bitcoin Gold should work with this Code part as well, because it is a direct Bitcoin Fork, and not Zcash based. I tested this for Globaltoken alone and with @sennevb who integrated GLT successfully into his Pool.

The stratum pull: https://github.com/s-nomp/node-stratum-pool/pull/20

If you have any more questions, let me know about it.
Cheers,

globaltoken committed:
Merge remote-tracking branch 'upstream/master'

globaltoken committed:
Added new GLT HF2 coin configs

globaltoken committed:
Use globaltoken snomp-stratum-pool

We use the GLT snomp-stratum-pool, because we use several changes for Multialgo.

globaltoken committed:
Fixed eh192 & mars name

globaltoken committed:
Added easyTXGeneration to config

zzzpotato commented:
This appears abandoned. Closing...
